Training_analysis.ipynb

```diff
@@ -638,7 +665,9 @@
     }
    ],
    "source": [
+    "#If using multiple workers comment out the first line and use the line with secondgroup=\"unique_episode\" in it\n",
     "simulation_agg = au.simulation_agg(df)\n",
+    "#simulation_agg = au.simulation_agg(df, secondgroup=\"unique_episode\")\n",
```

Comment:
Instead of commenting out a line it can be useful to define a variable that is then either None, blank or having a value.

Reply:
made multiple workers a var earlier in the file and use if else to handle throughout